controllers/snapshot/cache.go

```diff
@@ -54,8 +54,8 @@ func SetupApplicationCache(mgr ctrl.Manager) error {
 		"spec.environment", applicationIndexFunc)
 }
 
-// SetupEnvironmentCache adds a new index field to be able to search SnapshotEnvironmentBindings by Application.
-func SetupEnvironmentCache(mgr ctrl.Manager) error {
+// SetupSnapshotEnvironmentBindingCache adds a new index field to be able to search SnapshotEnvironmentBindings by Application.
+func SetupSnapshotEnvironmentBindingCache(mgr ctrl.Manager) error {
```

Comment:
Thanks for catching this!

Reply:
This is copied from your PR. Thanks for your finishing most of code!